kernel/configs/x86_64_kata_kvm_4.19.x

```diff
@@ -2600,7 +2600,13 @@ CONFIG_AUTOFS_FS=y
 CONFIG_FUSE_FS=y
 # CONFIG_CUSE is not set
 CONFIG_VIRTIO_FS=y
-# CONFIG_OVERLAY_FS is not set
+CONFIG_OVERLAY_FS=y
+CONFIG_OVERLAY_FS_REDIRECT_DIR=y
```

Comment:
Just to check - did you explicitly choose these settings, or take the menuconfig defaults?
Do we know we need/want `REDIRECT_DIR` for instance?

Reply:
I explicitly chose these, and explicitly disabled the =n ' s